src/index.html

```diff
@@ -0,0 +1,26 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<script src="https://kit.fontawesome.com/2b1476549e.js" crossorigin="anonymous"></script>
+    <title>To do</title>
+</head>
+<body>
+	<div class="box">
+		<div class="top-sec">
+			<h1>Today's Tasks</h1>
+			<button type="button" class="btn refresh"><i class="fas fa-sync-alt"></i></button>
+		</div>
+		<form>
+			<div class="input"><input id="input" class="inputField" type="text" placeholder="Add to your list here"><button type="button" class="btn enter"><i class='fa fa-arrow-left'></i></button></div>
```

Comment:
- [ ] It would be nice to add more lines for each of all the elements inside of this `div` for better readability. This means adding a different line for `input` and `button` 👍